resources/web/style/rtpcontainer.pcss

```diff
@@ -4,95 +4,179 @@
  * of styles with no clear use. We'll tame them when we get a better handle
  * on the rtpcontainer in general. */
 #rtpcontainer {
-  margin-bottom: 10px;
-  border-bottom: 1px solid #ccc;
-
-  h3 {
-    margin: 0 0 10px;
-    font-size: 26px;
-    font-weight: 300;
+  .mktg-promo {
+    border-radius: 5px;
+    margin-bottom: 40px;
+    padding: 20px;
+    border: 1px solid #ccc;
   }
-  ul {
+  ul.icons {
     padding-left: 2em;
     li {
-      list-style: disc;
-      padding: 0.15em 1em 0.15em 0;
-    }
-    &.icons {
+      align-items: center;
       list-style: none;
-      li {
-        display: flex;
-        align-items: center;
-        list-style: none;
-        padding-left: 0.6em;
-        min-height: 60px;
-        &:before {
-          content: "";
-          height: 2.5em;
-          width: 2.5em;
-          display: block;
-          float: left;
-          margin-left: -2.5em;
+      min-height: 60px;
+      padding-left: 0.6em;
+      display: flex;
+      :global(a.link) {
+        font-weight: normal;
+        margin-left: 1.2em;
+        display: block;
+        &:after {
           background-repeat: no-repeat;
-          background-color: #39BDB1;
-          background-size: 75%;
+          background-size: 70%;
           background-position: center center;
-          border-radius: 240px
-        }
-        a {
-          margin-left: 1.2em;
-          display: block;
-          &:after {
-            position: absolute;
-            margin-left: 0.65em;
-            content: inline("img/rtpcontainer/icon-expand.svg");
-            height: 0.7em;
-            width: 0.7em;
-            background-repeat: no-repeat;
-            background-size: 70%;
-            background-position: center center;
-          }
-        }
-
-        /* Product icons */
-        &.icon-elasticsearch-white:before {
-          background-image: inline("img/rtpcontainer/icon-elasticsearch-bb-white.svg");
```

Comment:
If we're not using any of these any more I think we should remove them from the docs repo.

Reply:
These background images will be used sometimes by rtp sometimes. They won't be used every time, but gives the people creating the rtp some options.